app/models/concerns/solr_document_extensions/work.rb

```diff
@@ -2,8 +2,8 @@
 module SolrDocumentExtensions::Work
   extend ActiveSupport::Concern
 
-  def artist
-    # TODO: needs to display an AIC.Agent
+  def artist_id
+    Array(self[Solrizer.solr_name('artist', :symbol)]).first
```

Comment:
why are you indexing ids in the solr_doc? wouldn't this just be the Agent pref_label?